src/course/course.module.ts

```diff
@@ -0,0 +1,17 @@
+import { Module, forwardRef } from '@nestjs/common';
+import { CourseService } from './course.service';
+import { CourseController } from './course.controller';
+import { InstructorModule } from 'src/instructor/instructor.module';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import { Course } from './entities/course.entity';
+
+@Module({
+  providers: [CourseService],
+  controllers: [CourseController],
+  imports: [
+    TypeOrmModule.forFeature([Course]),
+    forwardRef(() => InstructorModule),
```

Comment:
please dont use forwardRef, it has many edge case in the future, let create a repository file to use, as service-repository pattern
https://pmichaels.net/service-repository-pattern/#:~:text=What%20is%20the%20Service%20%2F%20Repository,var%20salesOrder%20%3D%20salesOrderService.